src/App.tsx

```diff
@@ -47,7 +55,7 @@ const queryClient = new QueryClient({
   },
 });
 
-const App: React.FunctionComponent = () => {
+const Root: React.FunctionComponent = () => {
```

Comment:
I don't think `Root` is actually needed any more after the migration - I would assume Layout would deal with rendering something. The contents of this though should probably be in `Layout` as right now none of it does anything.